### ISSUE TITLE
do not auto open database tree on launch

### ISSUE THREAD

**Issue by thewayur** [pawelsalawa/sqlitestudio#4028]
### Details
I do not want to see long tree of 50s database's table after launch. Even though i have grouped many dbs but its bit of annoying to **collapse them manually** before focusing on a particular db.
i have more than 200s of db listed. so please guide me to prevent it.
(thanks for this beautiful software)

i can try to recompile but i never did that before xD
 
### Operating system
windows

### SQLiteStudio version
3.3.0

**Comment by pawelsalawa** [comment]
You have 200 databases but you don't want to see them and don't want to collapse them manually.
What do you want then?

**Comment by thewayur** [comment]
> You have 200 databases but you don't want to see them and don't want to collapse them manually.
> What do you want then?

_sorry for my bad English and confusion._

1. i launched sqlitestudio
2. every previously connected dbs table is exapanded.

i do not want these connected dbs to be expanded by default.
only last launched i.e. previous state of dbs should be  expanded.

PIC1: AFTER LAUNCH (previously launched connected dbs)
![after launch](https://user-images.githubusercontent.com/2944767/110762593-f5694200-8276-11eb-9a74-18df888aa365.jpg)


pic2: i want them to stay collapsed after launching
![i want](https://user-images.githubusercontent.com/2944767/110762633-044ff480-8277-11eb-82a1-0fb44868dc68.jpg)


summary...
in short  i want sqlitestudio maintain db's expanded state after relaunch

RIGHT NOW EVERY CONNECTED DB is expanded on launch irrespective of their previous stated (collapsed or not)

hope i m making big trouble

**Comment by pawelsalawa** [comment]
Thank you. Now I understand completely.
Yeah... databases do expand upon connecting and since they are all re-connected when restarted application, they all get expanded. I will make it better. I'm not sure yet how, but I will.

**Comment by pawelsalawa** [comment]
I've resolved this by expanding only those of databases, which were expanded during application shutdown.

It basically restores the state/layout that you had before closing the app.
